1-Computer-Vision/Exercises/1-3-2-Stochastic-Gradient-Descent/2022-08-29-Stochastic-Gradient-Descent.ipynb

```diff
@@ -46,7 +46,7 @@
   },
```

Comment:
<p>Re:</p><blockquote>Looks like mean validation accuracy dropped slightly in your experiment run.</blockquote><p> </p><p>From this sample, I see no obvious changes to the GTSRB training images. Therefore, I wouldn't suspect any data corruption or issues with the image generator just yet. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jonathanloganmoran/ND0013-Self-Driving-Car-Engineer/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>